lib/Optimizer/Lower/Lower.cpp

```diff
@@ -448,6 +448,27 @@ static void lowerReluNode(Function *F, CompilationContext &cctx,
   replaceAllUsesOfWith(cctx.loweredInfoMap, R.getResult(), relu);
 }
 
+static void lowerHardSigmoidNode(Function *F, CompilationContext &cctx,
+                                 const HardSigmoidNode &R) {
+  LOG_SCOPE(F->getLogContext(), "lowerHardSigmoidNode")
+
+  auto ty = R.getResult().getType();
+  CHECK(ty->isFPType()) << "HardSigmoid: quantized type not supported: " << ty;
+
+  // max(0, min(1, alpha * x + beta))
+  auto *Alpha =
+      F->createSplat(DECORATE_NODE_NAME(R, "Alpha"), ty, R.getAlpha());
+  auto *Beta = F->createSplat(DECORATE_NODE_NAME(R, "Beta"), ty, R.getBeta());
```

Comment:
nit: Usually only capitalize letters for acronyms, otherwise `lowerCamelCase`.

```suggestion
  auto *alpha =
      F->createSplat(DECORATE_NODE_NAME(R, "Alpha"), ty, R.getAlpha());
  auto *beta = F->createSplat(DECORATE_NODE_NAME(R, "Beta"), ty, R.getBeta());
```

Reply:
handled in `Graph.cpp`
https://github.com/pytorch/glow/blob/301b91d8d2935cf2ac1d48517d18cdac655c3d59/lib/Graph/Graph.cpp#L1294-L1296